src/components/ToyForm.js

```diff
@@ -22,13 +22,15 @@ const ToyForm = ({ toy }) => {
     event.preventDefault();
     dispatch(createToy(form, toy));
     setForm({
-      ...form,
+      form,
```

Comment:
Take a look at what would happen in the setForm if you're setting `form` in the object.

First time setForm gets called, the `form` will look like:
```
{
  description: "",
  form: { description: "", imageUrl: "", name: "" },
  imageUrl: "",
  name: "my name"
}
```

The second time setForm gets called, `form `will look like:
```
{
  description: "description",
  form: {
    description: "description",
    form: { description: "", imageUrl: "", name: ""},
    imageUrl: "",
    name: "my name"
  },
  imageUrl: "",
  name: "my name"
}
```

...and so on.

Reply:
Ahhhh..... thank you. Right! I definitely do not want to do that.

Reply:
@Halezmarie Then why did you merge this bad code in?

Reply:
You should make changes to this branch until the issue is fixed, then re-request a review.  You should only be merging in code where a peer reviewer has approved it.  If this was production code, you just introduced a known bug into the code base (by merging into main, this would typically be auto-deployed to customers/users).

Reply:
I was thinking that the change requested would go through, but that isn't how it works haha. I am changing it now.

Reply:
You should be able to "revert" this merge, then work on fixing up the branch and reopening a PR (if this PR doesn't stay open) then re-request a review.